doc/classes/Viewport.xml

```diff
@@ -78,8 +78,8 @@
 				[b]Note:[/b] When trying to store the current texture (e.g. in a file), it might be completely black or outdated if used too early, especially when used in e.g. [method Node._ready]. To make sure the texture you get is correct, you can await [signal RenderingServer.frame_post_draw] signal.
 				[codeblock]
 				func _ready():
-				    await RenderingServer.frame_post_draw
-				    $Viewport.get_texture().get_image().save_png("user://Screenshot.png")
+				await RenderingServer.frame_post_draw
+				$Viewport.get_texture().get_image().save_png("user://Screenshot.png")
```

Comment:
That doesn't look intended.
(and is wrong)

Reply:
Bump .-.

Reply:
Gotcha. Weird, I Visual Studio refuses to let me intent those lines like that. Probably some non-sense setting. Had to open it in notepad ++ to make the change.